common/views/components/ArticleCard/ArticleCard.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import CompactCard from '../CompactCard/CompactCard';
+import Image from '../Image/Image';
+import type {Article} from '../../../model/articles';
+
+type Props = {|
+  article: Article,
+  showPosition: boolean
```

Comment:
I'm not sure I like passing this prop around, we've managed to avoid this mostly, but we need to have some sort of indicator that this card, in a specific, serial list, should show where's it is in that list.

I am also not massively convinced that this isn't going to change during QA, so might just leave it.

Reply:
Basically it's passing down the context of which list it is in.

We might need to do something similar when we have a list of 1 content type, and don't want to have massive repetition of labels.